static/style.css

```diff
@@ -4427,9 +4946,139 @@ blockquote .DocsMarkdown--header-anchor-positioner {
   color: var(--code-orange);
 }
 
-[theme="dark"] .DocsMarkdown--aside > p > a {
+[theme="dark"] .DocsMarkdown--aside>p>a {
   color: var(--orange);
 }
+
 pre {
   overflow: auto;
 }
+
+.wrapper {
+  width: 100%;
+  max-width: 700px;
+  margin: 0 auto;
+  position: relative;
+  padding-bottom: 5rem;
+  overflow-anchor: none;
+  font-family: var(--sans-serif-font-family);
+}
+
+.tabs {
+  position: relative;
+  border-radius: 3px;
+  margin-top: 3rem;
+  box-shadow: 0 1px rgba(var(--shadow-color-rgb), 0.12);
+}
+
+.tabs-content-wrapper {
+  height: 17rem;
+  background: rgba(var(--color-rgb), 0.05);
+}
+
+.tabs>ul {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  width: 100%;
+  outline: 1px solid rgba(var(--color-rgb), 0.09);
+  background-color: rgb(243 243 244);
+  border-radius: 4px;
+}
+
+[theme="dark"] .tabs>ul {
+  background-color: rgb(var(--accent-color-rgb));
+}
+
+.tabs>ul>li {
+  margin-bottom: 0;
+  border-bottom: 1px solid rgba(var(--color-rgb), 0.05);
+}
+
+.tabs::before,
+.tabs::after {
+  content: "";
+  display: flex;
+}
+
+.tabs::after {
+  clear: both;
+}
+
+.tab-switch {
+  display: none;
+}
+
+.tab-label-wrapper {
+  display: flex;
+  width: 25%;
+  align-items: center;
+  flex-direction: row;
+  margin-bottom: 0;
+  list-style-type: none;
+  box-shadow: 1px 0 rgba(var(--shadow-color-rgb), 0.12);
+}
+
+.tab-label-wrapper:hover {
+  background-color: rgba(var(--gray-3-rgb), 0.19);
+}
+
+[theme="dark"] .tab-label-wrapper:hover,
+[theme="dark"] .tab-label-wrapper a:hover {
+  background-color: rgba(var(--orange-rgb), 0.08);
+  color: var(--code-orange);
+}
+
+.tab-label {
+  position: relative;
+  white-space: nowrap;
+  height: 2.5rem;
+  padding: 0.6em 1.25em;
+  text-decoration: none;
+  text-align: center;
+  color: currentColor;
+  cursor: pointer;
+  top: 0;
+  font-weight: 700;
+  font-size: var(--code-font-size);
+  font-family: var(--monospace-font-family);
+}
+
+.tab {
```

Comment:
Looks good! Just a few changes to fix the positioning of the codeblock. 

```suggestion
.tab {
  max-height: 270px;
  display: none;
  color: #2c3e50;
  border-bottom: 0.25rem solid #bdc3c7;
  overflow: scroll;
}
```
<img width="665" alt="Screenshot 2022-06-30 at 15 13 06" src="https://user-images.githubusercontent.com/17781315/176687380-e47c6984-7960-43c2-8511-11fdc89deddb.png">

Reply:
Good catch with the padding! Thanks, Gift.